example/spec/distributionAPI/distributionSpec.js

```diff
@@ -19,16 +25,26 @@ const {
   uploadTestDataToBucket,
   deleteFolder
 } = require('../helpers/testUtils');
-const {
-  setDistributionApiEnvVars,
-  stopDistributionApi
-} = require('../helpers/apiUtils');
+const { setDistributionApiEnvVars } = require('../helpers/apiUtils');
 
 const config = loadConfig();
-const s3Data = [
-  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met'
-];
 
+const bucketsConfig = new BucketsConfig(config.buckets);
+const protectedBucketName = bucketsConfig.protectedBuckets()[0].name;
+const privateBucketName = bucketsConfig.privateBuckets()[0].name;
+const publicBucketName = bucketsConfig.publicBuckets()[0].name;
+process.env.stackName = config.stackName;
+const s3Data = [ '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met' ];
+
+/**
+ * Invoke the ApiDistributionLambda and return the headers location
+ * @param {filepath} filepath - request.path parameter
+ * @param {string} accessToken - authenticiation cookie (can be undefined).
+ */
+async function apiDistributionLocation(filepath, accessToken){
```

Comment:
This will probably be useful in other tests in the future. Maybe worth putting in the integration tests distribution helper file? Also, maybe re-name to `getDistributionApiRedirect`?

Reply:
Let's move it when we refactor.